client/resource_feature_store.go

```diff
@@ -193,6 +203,11 @@ func topicDestinationSchema() *schema.Resource {
 				Required:     true,
 				ValidateFunc: validation.StringIsNotWhiteSpace,
 			},
+			"format": {
+            			Type:     schema.TypeString,
+            			Required: true,
+				ValidateFunc: validation.StringIsNotWhiteSpace,
+            		},
```

Comment:
Formatting and validation. VScode should get this right on save, but gofmt should be run here.

Reply:
done